### PR TITLE
fix: latest fairos support

### PR DESCRIPTION
**Pull request by asabya**

### CODE REVIEW

File: src/constants.ts [hunk 1]
```diff
@@ -4,5 +4,5 @@
  */
 export const DEFAULT_BLOCKCHAIN_IMAGE = 'fairdatasociety/fdp-contracts-blockchain'
 export const DEFAULT_BLOCKCHAIN_CONTAINER = 'fdp-play-blockchain'
-export const DEFAULT_FAIROS_IMAGE = 'fairdatasociety/fairos-dfs:v0.7.3'
+export const DEFAULT_FAIROS_IMAGE = 'fairdatasociety/fairos-dfs:latest'
```
Comment (nugaon) [inline src/constants.ts:7]: latest is not necessary and it only makes the image name long
```suggestion
export const DEFAULT_FAIROS_IMAGE = 'fairdatasociety/fairos-dfs'
```

**Review by nugaon — APPROVED** [review]
Please use convential commits for the commit message format next time.
also check out my comment below

**Comment by crtahlin** [comment]
@asabya , what is needed here, what are the acceptance criteria?

**Comment by asabya** [comment]
> @asabya , what is needed here, what are the acceptance criteria?

If we are ok with the changes, we can merge this.

**Comment by nugaon** [comment]
Release [v2.0.2](https://github.com/fairDataSociety/fdp-play/releases/tag/v2.0.2) addresses this.

**Pull request merged**